### PR TITLE
Recreate event queue if it gets corrupt.

### PR DESCRIPTION
**Pull request by hamidp**

### CODE REVIEW

File: library/src/main/java/com/trello/snowman/EventQueue.java [hunk 1]
```diff
@@ -122,7 +127,43 @@ public void onRemove(ObjectQueue<Event> queue) {
...
+                }
+                catch (IOException e) {
+                    // This is really unrecoverable, we're pooped.
+                    throw new RuntimeException(e);
+                }
+            }
+        }
+    }
+
+    private void tryResetQueueFile() {
```
Comment (dlew) [inline library/src/main/java/com/trello/snowman/EventQueue.java]: Minor, but the name of this method seems misleading.

**Comment by dlew** [comment]
:+1: Besides the minor point.

**Comment by hamidp** [comment]
@dlew How about now?

**Comment by dlew** [comment]
@hamidp :+1: LGTM

**Pull request merged**